src/lib/components/Question.svelte

```diff
@@ -90,16 +106,16 @@
 								</div>
 							</Dialog.Content>
 						{:else}
-							<Dialog.Content class="w-80 rounded-xl">
-								<Dialog.Title class="mt-4">Answer all mandatory questions!</Dialog.Title>
-								<Dialog.Description>
-									Please make sure you have answered all the mandatory questions to submit the test.
-								</Dialog.Description>
-
-								<Dialog.Close><Button class="mt-2 w-full">Okay</Button></Dialog.Close>
-							</Dialog.Content>
+							{@render mandatoryQuestionDialog(true)}
 						{/if}
 					</Dialog.Root>
+				{:else if !answeredCurrentMandatory(paginationPage, perPage, selectedQuestions, questions)}
+					<Dialog.Root>
+						<Dialog.Trigger>
+							<Pagination.NextButton disabled={true} />
```

Comment:
@ninad-ict 

Let's not disable the next button as we are prompting the user about mandatory questions.